Library/Homebrew/requirements/python_requirement.rb

```diff
@@ -21,6 +21,7 @@ class PythonRequirement < Requirement
     # Homebrew Python should take precedence over older Pythons in the PATH
     elsif short_version != Version.create("2.7")
       ENV.prepend_path "PATH", Formula["python"].opt_bin
+      ENV.prepend_path "PATH", Formula["python"].libexec/"bin"
```

Comment:
should it be `.opt_libexec/"bin"` for the sake of consistency with the `.opt_bin` above it?